ginga/rv/plugins/Contents.py

```diff
@@ -39,6 +39,22 @@
 make it so by setting the "closeable" setting to True in the configuration
 file--then Close and Help buttons will be added to the bottom of the UI.
 
+** Excluding images from Contents **
```

Comment:
Remove whitespace to correctly render it as bold font.
```suggestion
**Excluding images from Contents**
```